.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,16 @@
 <!--
 Fill in the relevant information below to help triage your issue.
+
 Pick the target branch based on the following criteria:
   * Documentation improvement: master branch
   * Bugfix: master branch
   * QA improvement (additional tests, CS fixes, etc.) that does not change code
     behavior: master branch
   * New feature, or refactor of existing code: develop branch
+
+You MUST use the --signoff or -s flag when using "git commit" for us to be able
+to accept your patch; please see the project contributing guide and
+https://developercertificate.org for details.
```

Comment:
I think the focus on the flag places wrong emphasis.
```suggestion
You MUST provide a signoff in your commits; use the --signoff or -s flag when
using "git commit" for us to be able to accept your patch; please see the
project contributing guide and https://developercertificate.org for details.

```

Reply:
Thanks for this feedback; I've reworded it.